images/rlimit.proto

```diff
@@ -1,6 +1,9 @@
 syntax = "proto2";
 
+import "opts.proto";
+
 message rlimit_entry {
 	required uint64		cur = 1;
 	required uint64		max = 2;
+	optional uint32		rlimit_type = 3 [(criu).dict = "rlimit"];
```

Comment:
This field isn't required to restore a process, so it should not be in the image file. You can use an array index to determine a type.